### PR TITLE
Clarify which node component(s) contact control plane

### PR DESCRIPTION
**Pull request by sravyasambaturu**
PR related to https://github.com/kubernetes/website/pull/39695

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h2 style="color: #3771e3;">Cluster Diagram</h2>
 
         <div class="row">
             <div class="col-md-8">
-                <p>When you deploy applications on Kubernetes, you tell the control plane to start the application containers. The control plane schedules the containers to run on the cluster's nodes. <b>The nodes communicate with the control plane using the <a href="/docs/concepts/overview/kubernetes-api/">Kubernetes API</a></b>, which the control plane exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p>
+                <p>When you deploy applications on Kubernetes, you tell the control plane to start the application containers. The control plane schedules the containers to run on the cluster's nodes. <b>The kubelet on each node communicates with the control plane using the <a href="/docs/concepts/overview/kubernetes-api/">Kubernetes API</a></b>, which the control plane exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p>
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:89]: I think this is a better replacement:
```suggestion
                <p>When you deploy applications on Kubernetes, you tell the control plane to start the application containers. The control plane schedules the containers to run on the cluster's nodes. <b>Node-level components, such as the kubelet, communicate with the control plane using the <a href="/docs/concepts/overview/kubernetes-api/">Kubernetes API</a></b>, which the control plane exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p>
```

the kubelet is not the only component on a node that might contact the control plane

**Comment by sftim** [comment]
/retitle Clarify which node component(s) contact control plane

File: content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html (hunk 1 above)
Comment (utkarsh-singh1) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html]: Hi @sftim, maybe we can mention this [link](https://kubernetes.io/docs/concepts/overview/components/#node-components), under Node-level components.

Comment (sftim) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html]: > Maybe we can mention this [link](https://kubernetes.io/docs/concepts/overview/components/#node-components), under Node-level components.

This is true. You might also want to look at https://github.com/kubernetes/website/issues/29049#issuecomment-885977653 and the issue that this comment is part of.

**Pull request merged**